app/uk/gov/hmrc/eusubsidycompliancefrontend/connectors/EscConnector.scala

```diff
@@ -175,7 +146,7 @@ class EscConnectorImpl @Inject()(http: HttpClient,
     )
   }
 
-  private def toSubsidyDelete(nonHmrcSubsidy: NonHmrcSubsidy, undertakingRef: UndertakingRef): SubsidyUpdate = {
+  private def toSubsidyDelete(nonHmrcSubsidy: NonHmrcSubsidy, undertakingRef: UndertakingRef) =
```

Comment:
Think you may have removed typing by mistake

Reply:
It doesn't seem to be needed since it's inferred correctly. I tend to only add explicit typing on public members. I can add it back if you feel it's needed?